Frameworks/UIKit/UIImage.mm

```diff
@@ -494,11 +494,16 @@ - (void)drawAtPoint:(CGPoint)point blendMode:(CGBlendMode)mode alpha:(float)alph
     pos.size.height = (img_width / _scale);
 
     srcRect.origin.x = 0;
-    srcRect.origin.y = img_height;
+    srcRect.origin.y = 0;
     srcRect.size.width = img_width;
-    srcRect.size.height = -img_height;
+    srcRect.size.height = img_height;
```

Comment:
Actually, since this dropped the use of DrawImageRect, srcRect can be removed entirely.